libraries/cbor/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,20 @@
+#![no_main]
+#[macro_use] 
+extern crate libfuzzer_sys;
+extern crate cbor;
+extern crate alloc;
+
+use alloc::vec::Vec;
+
+fuzz_target!(|data: &[u8]| {
+    let encoded = cbor::read(data);
+    match encoded{
```

Comment:
There is a space missing here compared to our usual style. Usually an automated check would run `cargo fmt` to ensure correct code style. It seems that our github workflow does not cover your fuzzing code. Can you please add this path and fix the style?

Reply:
Sorry for triggering the error, just wanted to make sure if I added the path correctly and it can be detected.